commerce_coordinator/apps/lms/views.py

```diff
@@ -85,7 +86,7 @@ def _redirect_response_payment(self, request):
 
         get_items = list(self.request.GET.lists())
         redirect_url = None
-        if "bundle" in dict(get_items):
+        if "bundle" in dict(get_items) and not is_program_redirection_to_ct_enabled(request):
```

Comment:
aren't we checking this inside the pipeline. I think we can remove this code and just run pipeline. no?

Reply:
Yes, we are checking this inside the pipeline. However, it's currently behind a feature flag. Once the feature flag is removed, we can delete this code, and only the pipeline logic will run.

Reply:
makes sense